src/components/common/Title_Styled.js

```diff
@@ -0,0 +1,49 @@
+import { className } from "postcss-selector-parser";
+import styled, { css } from "styled-components";
+
+export const Title = styled.h1`
```

Comment:
A good practice is to name the file and the component using the same name, in this case the file is named `Title_Styled.js` and the component is named `Title`. Also, the files that contain a component are usually named using pascal case, this is not mandatory or universal but is pretty common in react